src/components/melding/LukkbarNyMelding.tsx

```diff
@@ -52,35 +53,37 @@ export function LukkbarNyMelding() {
     return (
         <Panel onResize={onExpand} ref={panelRef} defaultSize={PANEL_SIZE} minSize={20} maxSize={60} order={2}>
             <VStack height="100%" gap="2">
-                <UbesvarteMeldinger />
-                <TildelteOppgaver />
-                <SendMelding
-                    lukkeKnapp={
-                        <HStack gap="2">
-                            <Button
-                                aria-hidden
-                                type="button"
-                                icon={isLarge ? <ShrinkIcon title="Minimer" /> : <ExpandIcon title="Ekspander" />}
-                                variant="tertiary"
-                                size="small"
-                                onClick={() => {
-                                    if (!panelRef.current) return;
+                <ErrorBoundary boundaryName={'sendmelding'}>
```

Comment:
```suggestion
                <ErrorBoundary boundaryName="sendmelding">
```

Reply:
Burde egt. legge inn en rule i biome for å fikse disse.

Reply:
https://biomejs.dev/linter/rules/use-consistent-curly-braces/

Reply:
La til regel med warning. Eksisterande feil blei ikkje retta opp med lint fix då det er unsafe fixes. Skal eg kjøre kommandoen uansett? @LudvigHz